### ISSUE TITLE
GitlabCI image: hadolint/hadolint:2.3.0

### ISSUE THREAD

**Issue by firecow** [hadolint/hadolint#616]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
![image](https://user-images.githubusercontent.com/973602/116357518-d3af2480-a7fc-11eb-9bdb-dc0c41cb8705.png)

### Actual behavior
![image](https://user-images.githubusercontent.com/973602/116357780-2dafea00-a7fd-11eb-9f31-a3a834dfa135.png)

### Steps to reproduce the behavior
```yml 
# .gitlab-ci.yml
--- 
# @Description Hadolint Dockerfile
hadolint:
  image: hadolint/hadolint:2.3.0
  script:
    - hadolint Dockerfile
```

```bash
gitlab-ci-local hadolint
```

**Dockerfile**

```Dockerfile

# hadolint ignore=DL3007
FROM koalaman/shellcheck:latest AS shellcheck

FROM python:3-alpine AS python-base

# Update 'Utilities' section in ./README.md when adding utils here.
# Ignore non-pinned versions. We want the newest.
RUN apk add --no-cache \
        bash \
        dumb-init \
        curl \
        jq \
    && \
    pip install --no-cache-dir --disable-pip-version-check \
        gixy \
        j2cli \
        yamllint \
        yq

WORKDIR "/app"

COPY --from=shellcheck /bin/shellcheck /usr/local/bin/shellcheck

# Ensure: Reap of zombie processes.
ENTRYPOINT ["dumb-init", "--"]

# Output something useful on missing command.
CMD ["bash", "-c", "echo 'Run with a command! See https://gitlab.cego.dk/cego/dockerized-utilities' >&2; exit 1"]

```

I believe you've started using `FROM busybox` or `FROM scratch`, which doesn't play well with gitlab-ci, can you guys be persuaded to use `FROM alpine` perhaps ?

**Comment by firecow** [comment]
![image](https://user-images.githubusercontent.com/973602/116358383-e1b17500-a7fd-11eb-9118-ccc5c9fbf460.png)

On another note, i'm an idiot !!!! :smile:
